ios/Gekidou/Sources/Gekidou/PushNotification/PushNotification.swift

```diff
@@ -75,9 +75,14 @@ public class PushNotification: NSObject {
                 url, withMethod: "POST", withBody: jsonData,
                 andHeaders: headers, forServerUrl: ackNotification.serverUrl) {[weak self] data, response, error in
                     if error != nil && ackNotification.isIdLoaded,
+                       let nsError = error as? NSError,
                        let fibonacciBackoffsInSeconds = self?.fibonacciBackoffsInSeconds,
                        let retryIndex = self?.retryIndex,
                        fibonacciBackoffsInSeconds.count > retryIndex {
+                        if nsError.code == -999 {
```

Comment:
Can you reference the constant so it's easier to read? i.e. no magic numbers

https://developer.apple.com/documentation/foundation/1508628-url_loading_system_error_codes/nsurlerrorcancelled